homeassistant/components/glances/sensor.py

```diff
@@ -272,7 +272,9 @@ def _migrate_old_unique_ids(
             for disk in coordinator.data[description.type]:
                 _migrate_old_unique_ids(
                     hass,
-                    f"{coordinator.host}-{name} {disk['mnt_point']} {description.name_suffix}",
+                    (
+                        f"{coordinator.host}-{name} {disk['mnt_point']} {description.name_suffix}"
+                    ),
```

Comment:
The changes in this file are not helpful IMO, we should just drop them.

Reply:
[cf642e0](https://github.com/home-assistant/core/pull/84393/commits/cf642e0030fc74b1ae5d96312ab91bd2c7a23702)